lib/init/index.js

```diff
@@ -1,13 +1,91 @@
-export function builder(yargs) {
+import { copy, existsSync } from 'fs-extra';
+import { join } from 'path';
+import chalk from 'chalk';
+import cleanupFiles from './utils/clean-up-files';
+
+const CWD = process.cwd();
+const BIGTEST_DIR = `${CWD}/bigtest`;
+let CLI_TEMPLATE_DIR = join(__dirname, '../../templates');
+let pathExists = path => existsSync(path);
```

Comment:
This kinda makes it a misleading if you're reading further down and think `pathExists` is normally async. Let's just stick to calling it `existsSync`.